doc/content/source/tallies/CellTally.md

```diff
@@ -0,0 +1,33 @@
+# CellTally
+
+!alert note
+`CellTally` can only be added to problems when the input files contains a [OpenMCCellAverageProblem](OpenMCCellAverageProblem.md)
+in the `[Problem]` block. Otherwise, attempting to add this tally will result in an error.
+
+## Description
+
+The `CellTally` class wraps an OpenMC tally with a distributed cell filter to enable data transfer
+from CSG cells to a [MOOSEMesh](https://mooseframework.inl.gov/source/mesh/MooseMesh.html) mirror
+of the OpenMC geometry. The cell to element mapping established by the [OpenMCCellAverageProblem](OpenMCCellAverageProblem.md)
+is used to facilitate this data transfer to the mesh mirror. When using a `CellTally`, `tally_blocks`
+specifies which blocks in the `[Mesh]` should be tallied. Then, any OpenMC cells that map to those
+blocks are added to a cell tally, with one bin for each unique cell ID/instance combination.
+`check_equal_mapped_tally_volumes` is used to confirm whether the elements mapped to a specific cell
```

Comment:
This explanation isn't quite correct - this parameter doesn't check if the mapped element volume matches the CSG cell volume. Instead, it checks that if we find 99 CSG cells to tally, that they all map to the same volume on the MOOSE side.